.github/scripts/verify_changes.sh

```diff
@@ -17,15 +25,32 @@ else
   base_commit=$(git log origin/$ref_name --oneline | head -2 | awk 'NR==2 {print $1}')
 fi
 
-# git diff with ... shows the differences count between base_commit and head_commit starting at the last common commit
-change_count=$(git diff $base_commit...$head_commit --name-only | awk -F"/" '{ print $1}' | uniq | wc -l)
+# git diff with ... shows the differences between base_commit and head_commit starting at the last common commit
+changed_dir=$(git diff $base_commit...$head_commit --name-only | awk -F"/" '{ print $1}' | uniq)
+change_count=$(echo "$changed_dir" | wc -l)
```

Comment:
echo will add a newline when echoing an empty string, which means that even if the git diff is empty, the change_count will be 1:
```sh
$  i=$(git diff --name-only | awk -F"/" '{ print $1}' | uniq) echo $i | wc -l
       1
$  git diff --name-only | awk -F"/" '{ print $1}' | uniq | wc -l
```

I don't think this is a problem, given how the if conditions are written, but it's something to be aware of.

Reply:
Good call. I should fix this.